_posts/2016-09-14-supercharged-seo-with-drupal-8.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: "Supercharged SEO with Drupal 8"
+date: 2016-09-14
+author: Oksana Cyrwus
+tags: seo drupal drupal8
```

Comment:
How about adding `SEO` as a tag, or perhaps `marketing` since SEO may be too specific to be used frequently enough in future posts